modules/travel_pay/spec/services/client_spec.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe TravelPay::Client do
+  let(:veis_params_stub) do
+    {
+      client_id: 'sample_id',
+      client_secret: 'sample_client',
+      client_info: 1,
+      grant_type: 'client_credentials',
+      resource: 'sample_resource'
+    }
+  end
+
+  before do
+    @stubs = Faraday::Adapter::Test::Stubs.new
+    conn = Faraday.new { |b| b.adapter(:test, @stubs) }
+    allow_any_instance_of(TravelPay::Client).to receive(:connection).and_return(conn)
+    allow_any_instance_of(TravelPay::Client).to receive(:veis_params).and_return(veis_params_stub)
```

Comment:
`any_instance` [appears to be on its way out](https://www.rubydoc.info/gems/rubocop-rspec/1.7.0/RuboCop/Cop/RSpec/AnyInstance). How feasible is it to use an instance double here? Potentially some constructor injection?